test/functional/tag_controller_test.rb

```diff
@@ -117,15 +133,15 @@ def setup
   end
 
   test 'tag search' do
-    get :index , :search => "featured"
+    get :index , params: { :search => "featured" }
```

Comment:
Ah, could we switch this to `search: "featured"` ?

Reply:
Yeap... Rubocop fixes this automatically if we ask it to. Is it possible to have it on CI?

Reply:
I think so, I haven't configured it before. Can you open an issue for it and we can find the next steps? I want to know how autoformatting works, does it wait until the last step or could it require contributors to pull back down the formatting fixes? Let's figure it out.